xml/dm_registers.xml

```diff
@@ -435,8 +434,7 @@
       hold identifier information which is not
       further specified in this document.
 
-      The Device Tree itself is described in the RISC-V Privileged
-      Specification.
+      The Device Tree itself is described in the Privileged Spec.
```

Comment:
Is it? It may have been removed.

Reply:
It's mentioned, but not defined anywhere. Instead it talks about a more handwavy "configuration string" and specifically mentions that platforms can provide a "configuration string pointer." I'll rename this register to reflect that.

Reply:
I thought we changed this to Device tree FROM config string because they were going to get rid of Config string... weird.

Reply:
From the preface to Version 1.9.1:
> Change configuration string proposal to be use a search process that supports various formats including Device Tree String and flattened Device Tree.

I don't know if I went and looked at the language when I changed the debug spec to refer to Device Tree.